src/BookReader/BookModel.js

```diff
@@ -352,8 +352,9 @@ export class PageModel {
    * @param {PageIndex} index
    */
   constructor(book, index) {
-    // TODO: Get default from config
-    this.ppi = book._getDataProp(index, 'ppi', book.ppi);
+    // Values less than 10 cause the UI to not work correctly
+    const pagePPI = book._getDataProp(index, 'ppi', book.ppi);
+    this.ppi = Math.max(pagePPI < 10 ? book.ppi : pagePPI, 10);
```

Comment:
👍 q: what does pagePPI default to?

Reply:
Oh the default is the last argument to getDataProp, book.ppi . This is the value in the metadata for that entire book